### PR TITLE
Bigz/funding clamp

### PR DESCRIPTION
**Pull request by 0xbigz**
- max oracle price update to twap is 20% from twap
- clamp funding rate to 2/24 %

### CODE REVIEW

File: programs/clearing_house/src/math/amm.rs [hunk 1]
```diff
@@ -102,9 +102,38 @@ pub fn update_oracle_price_twap(
...
-    amm.last_oracle_price_twap = oracle_price_twap;
-    amm.last_oracle_price_twap_ts = now;
+    let new_oracle_price_spread = oracle_price
+        .checked_sub(amm.last_oracle_price_twap)
+        .ok_or_else(math_error!())?;
+
+    // cap new oracle update 
+    let oracle_price_20pct = oracle_price.checked_div(5).ok_or_else(math_error!())?;
+
+    let new_oracle_update_price =
```
Comment (crispheaney) [inline programs/clearing_house/src/math/amm.rs:112]: would call this `capped_oracle_update_price`

File: programs/clearing_house/src/math/amm.rs [hunk 2]
```diff
@@ -102,9 +102,38 @@ pub fn update_oracle_price_twap(
     now: i64,
     oracle_price: i128,
 ) -> ClearingHouseResult<i128> {
-    let oracle_price_twap = calculate_new_oracle_price_twap(amm, now, oracle_price)?;
-    amm.last_oracle_price_twap = oracle_price_twap;
-    amm.last_oracle_price_twap_ts = now;
+    let new_oracle_price_spread = oracle_price
```
Comment (crispheaney) [inline programs/clearing_house/src/math/amm.rs:105]: would call this capped_oracle_price_spread

File: test-scripts/run-anchor-tests.sh [hunk 3]
```diff
@@ -5,6 +5,7 @@ if [ "$1" != "--skip-build" ]
 fi
 
 test_files=(clearingHouse.ts pyth.ts userAccount.ts admin.ts updateK.ts adminWithdraw.ts curve.ts whitelist.ts fees.ts idempotentCurve.ts maxDeposit.ts deleteUser.ts maxPositions.ts maxReserves.ts roundInFavor.ts minimumTradeSize.ts cappedSymFunding.ts)
+test_files=(cappedSymFunding.ts)
```
Comment (crispheaney) [inline test-scripts/run-anchor-tests.sh:8]: should remove

Comment (0xbigz) [reply test-scripts/run-anchor-tests.sh]: just pushed a cleanup :)

**Pull request merged**